### PR TITLE
Implement mithril client wasm in explorer

### LINKED ISSUE #1409: Light Wallet: Implement the mithril client WASM package in the explorer
## Why
We want to add the Mithril certificate verification in the explorer.

## What
Use the Mithril client WASM to verify a certificate and its chain in the browser form the explorer. We have already made a first implementation in this PoC [#1254](https://github.com/input-output-hk/mithril/issues/1254#issuecomment-1744702606)

## How

- [x] Add a popup (or link to a new page) from the Certificate popup
- [x] Compute automatically the genesis verification key from the url of the aggregator (by using the network code)
- [x] Adapt the npm package publication (bundler or not, CI, ...)

## Later
- How to cache the certificates already verified by the browser
- Handle abort of `verify_certificate_chain` function call.

### PR DESCRIPTION
**Pull request by dlachaumepalo**
## Content
This PR includes the implementation of `mithril-client-wasm` library in the Mithril explorer. It adds a new feature to verify certificates using the WASM library.

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [x] Crates versions are updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Comments
- `wasm-pack build` command in the Makefile of the `mithril-client-wasm` lib had to be modified to add the `--target web` option. This greatly simplifies the configuration for Next.js projects. Commands executed to build the library in the CI have also been updated.
- The CI's actions had to be reworked. The contents of the file `docs.yml` have been moved to the file `ci.yml`. This has been done to handle the `yarn install` step in the build-test-explorer job.
This job needs `mithril-client-wasm` to be built and the artifact published to install the dependency in the `mithril-explorer` project.

## Issue(s)
Closes #1409

### CODE REVIEW

File: mithril-explorer/src/utils.js [hunk 1]
```diff
@@ -78,6 +78,20 @@ function getCExplorerUrlForPool(network, partyId) {
   return url;
 }
 
+function calculatesProcessDuration(startTime) {
```
Comment (jpraynaud) [inline mithril-explorer/src/utils.js:81]: Maybe name it differently:
```suggestion
function formatProcessDuration(startTime) {
```

File: mithril-explorer/src/utils.js [hunk 2]
```diff
@@ -78,6 +78,20 @@ function getCExplorerUrlForPool(network, partyId) {
...
+  const duration = performance.now() - startTime;
+  const minutes = Math.floor(duration / 60000);
+  const seconds = Math.floor((duration % 60000) / 1000);
+  return(
+    minutes > 0 ? `${minutes} minutes and ${seconds} seconds` : `${seconds} seconds`
+  );
+}
+
+function retrieveAggregatorNetworkFromUrl(aggregatorUrl) {
```
Comment (jpraynaud) [inline mithril-explorer/src/utils.js:90]: Maybe name it differently:
```suggestion
function computeAggregatorNetworkFromUrl(aggregatorUrl) {
```

**Review by jpraynaud — APPROVED** [review]
LGTM :+1: 

I left few comments below

File: mithril-client-wasm/www/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 // TODO: improve the management of function returned values by adding an event mechanism
 // Errors should be handled too.
 
-import { MithrilClient } from "@mithril-dev/mithril-client-wasm"
+import init, { MithrilClient } from "@mithril-dev/mithril-client-wasm"
```
[the remark this answers is not in the document]
Comment (dlachaumepalo) [reply mithril-client-wasm/www/index.js:4]: It's already done in the in the most recent commits.

**Review by ghubertpalo — APPROVED** [review]

**Pull request merged**